### PR TITLE
[#136] Add google maps integration

### LINKED ISSUE #136: Add real map to the footer
## Google maps integration with Hacklag website
 * Map should show Hacklag foundation on google maps located in footer

### PR DESCRIPTION
**Pull request by kamilant**
## Google maps integration
 * Added google maps react component
 * Added google maps to the footer
 * Added hackbat marker 
resolve #136

### CODE REVIEW

File: config/eslint.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
   parser: 'babel-eslint',
   extends: 'airbnb',
   globals: {
+    GOOGLE_MAPS_API_KEY:true,
```
Comment (kkotelczuk) [inline config/eslint.js]: (code style): Missing space

File: src/bytes/Footer/Footer.js [hunk 2]
```diff
@@ -62,6 +61,26 @@ const Footer = ({
...
+        defaultZoom={17}
+      >
+        <div
+          lat={53.1311871}
+          lng={23.1581426}
+        >
+          <img
+            width={'100px'}
+            role={'presentation'}
+            src={'http://i.imgur.com/fYrxDeK.png'}
```
Comment (kkotelczuk) [inline src/bytes/Footer/Footer.js]: Why did you use `Imgur` instead of an image stored in `assets` folder?

**Review by kkotelczuk — COMMENTED** [review]
Next time, please add also screenshot to show your changes.

**Comment by kamilant** [comment]
screen shot 2017-02-25 at 00 11 54

**Review by kkotelczuk — APPROVED** [review]

**Review by Idered — APPROVED** [review]

**Pull request merged**